relay-metrics/src/aggregation.rs

```diff
@@ -1811,37 +1828,41 @@ impl AggregatorService {
 
         let mut stats = BTreeMap::new();
 
-        relay_statsd::metric!(timer(MetricTimers::BucketsScanDuration), {
-            let bucket_interval = self.config.bucket_interval;
-            let cost_tracker = &mut self.cost_tracker;
-            self.buckets.retain(|key, entry| {
-                if force || entry.elapsed() {
-                    // Take the value and leave a placeholder behind. It'll be removed right after.
-                    let value = mem::replace(&mut entry.value, BucketValue::Counter(0.0));
-                    cost_tracker.subtract_cost(key.project_key, key.cost());
-                    cost_tracker.subtract_cost(key.project_key, value.cost());
-
-                    let (bucket_count, item_count) = stats
-                        .entry((metric_type_tag(&value), metric_name_tag(&key.metric_name)))
-                        .or_insert((0usize, 0usize));
-                    *bucket_count += 1;
-                    *item_count += value.len();
-
-                    let bucket = Bucket::from_parts(key.clone(), bucket_interval, value);
-                    buckets
-                        .entry(key.project_key)
-                        .or_default()
-                        .push(HashedBucket {
-                            hashed_key: key.hash64(),
-                            bucket,
-                        });
-
-                    false
-                } else {
-                    true
-                }
-            });
-        });
+        relay_statsd::metric!(
```

Comment:
really wish github would show it in another color if only the indentation changed